cypress-io/cypress
# fix: only render debug artifacts when info is available

- Closes #25319

### User facing changelog
n/a - Feature branch

### Additional details
Only render debug artifacts if the corresponding `has<attr>` is true and the `<attr>Url` is defined.

### Steps to test

1. Check out this branch and the cypress-services feature branch, make sure the cypress-services dev is running
2. Create a project if you don't have one
3. Choose a repo with Cypress setup and a simple test that will fail
4. Add `video: false` to the cypress config
5. Run `CYPRESS_INTERNAL_ENV=development npx cypress run --record --key <RECORD_KEY>`
6. Edit `DebugContainer.vue` and change `runByNumber(runNumber: <RUN_NUMBER>)`, the last step should have recorded a run to your locally running dashboard and you can find the run number there

You can disable screenshots if you like and record another run.

### How has the user experience changed?

https://user-images.githubusercontent.com/25158820/210644704-ebc0c237-1285-48c9-8fdb-09e2e0f37ef9.mov

### PR Tasks
